### DISCUSSION TITLE
Is anyone getting Ads on the new outlook (preview) (even thought this tool is supposed to activate office 365), so normally i wouldnt be getting any ads

### DISCUSSION THREAD

**Discussion by GitExploraa** [YerongAI/Office-Tool#657] [Q&A]
Or is the new preview not part of the cracked apps or something?

**Comment by YerongAI** [comment]
It's better to take a screenshot about the AD.

**Comment by GitExploraa** [comment]
this one for example

![image](https://user-images.githubusercontent.com/44654056/230793019-5621bccd-9581-4826-8e79-6755dbda925b.png)

**Comment by YerongAI** [comment]
> this one for example
> 
> ![image](https://user-images.githubusercontent.com/44654056/230793019-5621bccd-9581-4826-8e79-6755dbda925b.png)
> 

I don't think it's an AD from Outlook, it just means the mail you have received looks like an AD.

**Reply by GitExploraa** [reply]
i'm pretty sure it is, just see on the corner it shows "Ad", plus i have checked online that this is how Outlook displays the Ads, inside the inbox as if they were emails... happened quite a few of times already, ill update this if anything new comes, Thank you

**Reply by YerongAI** [reply]
> i'm pretty sure it is, just see on the corner it shows "Ad", plus i have checked online that this is how Outlook displays the Ads, inside the inbox as if they were emails... happened quite a few of times already, ill update this if anything new comes, Thank you

It's Outlook Web? It doesn't look like a desktop app.

**Reply by GitExploraa** [reply]
At its core yeah it's based on a PWA from what I've read, but i got it enabled through the outlook app that gets installed with office (by clicking the "try the new outlook" toggle on the top), maybe it being a PWA is the reason why it doesn't get patched by this tool?

**Reply by YerongAI** [reply]
> At its core yeah it's based on a PWA from what I've read, but i got it enabled through the outlook app that gets installed with office (by clicking the "try the new outlook" toggle on the top), maybe it being a PWA is the reason why it doesn't get patched by this tool?

Fine, you should know the following things:

1. PWA is a web based app, so no one can modify or crack it.
2. Our tool can activate the desktop apps, includes Outlook. But you didn't use it, you are using PWA.
3. To fix the problem, please use desktop app instead. Or accept the ADs.

**Reply by GitExploraa** [reply]
I didn't know how the app worked at first, and i also updated it as part of the normal office install (like with the toggle to activate the new design of word for example), Thanks for replying either way and thanks for this amazing tool
